README.md

```diff
@@ -15,10 +15,16 @@ Learn more about [cloud.gov](https://cloud.gov).
 ## Tech Stack
 - `Go` (v1.6.2 required) for the backend server. [![Go Code Coverage Status](https://coveralls.io/repos/18F/cg-deck/badge.svg?branch=master&service=github)](https://coveralls.io/github/18F/cg-deck?branch=master)
 
-- `AngularJS` for the frontend. [![JS Code Coverage Status](http://codecov.io/github/18F/cg-deck/coverage.svg?branch=master)](http://codecov.io/github/18F/cg-deck?branch=master)
-
+- `Node` (v.4.2.3) and `React` for the frontend. [![JS Code Coverage Status](https://coveralls.io/repos/18F/cg-deck/badge.svg?branch=staging-alpha&service=github)](https://coveralls.io/repos/18F/cg-deck/badge.svg?branch=staging-alpha&service=github)
 
 ## Setup
+### Cloning the repository
+If you are unfamiliar with `Go` project directory structure, you want the code in this repository to be in something like `<your-code-directory>/cg-deck-ws/src/github.com/18f/cg-deck`. You can use that exact pattern by cloning the repository with:
+
+```
+git clone git@github.com:18F/cg-deck.git cg-deck-ws/src/github.com/18F/cg-deck
+```
+
 ### Create a Client with UAAC
```

Comment:
I think this is still necessary, I just gave you my generated client tokens when we were getting your environment setup. In fact, if you could test out these instructions to make sure they work that would be really valuable.